### PR TITLE
flexoptix-app: 5.11.0 -> 5.12.1

### LINKED ISSUE #183671: flexoptix-app is outdated
###### Checklist
- [x] Checked the [nixpkgs master branch](https://github.com/NixOS/nixpkgs)
- [x] Checked the [nixpkgs pull requests](https://github.com/NixOS/nixpkgs/pulls)

###### Project name
`nix search` name:
flexoptix-app

current version: 5.11.0
desired version: 5.12.1

###### Notify maintainers
maintainers: @dasJ

###### Note for maintainers

Please tag this issue in your PR.

### PR DESCRIPTION
**Pull request by dasJ**
###### Description of changes

Closes #183671

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.11 Release Notes (or backporting 22.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2211-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

See the version in the top-right menu:
image

### CODE REVIEW

File: pkgs/tools/misc/flexoptix-app/default.nix [hunk 1]
```diff
@@ -1,12 +1,12 @@
 { lib, appimageTools, fetchurl, nodePackages }: let
   pname = "flexoptix-app";
-  version = "5.11.0";
+  version = "5.12.1";
   name = "${pname}-${version}";
```
Comment (mweinelt) [inline pkgs/tools/misc/flexoptix-app/default.nix:4]: That looks only slightly redundant.

Comment (dasJ) [reply pkgs/tools/misc/flexoptix-app/default.nix:4]: Not sure what you mean. I let both name and pname because I need both later

Comment (mweinelt) [reply pkgs/tools/misc/flexoptix-app/default.nix:4]: Hrm. The `name` attribute is already exactly that concat, but it is not available within the derivation?

Comment (dasJ) [reply pkgs/tools/misc/flexoptix-app/default.nix]: Removed it as all appimage things now also take `pname`

**Pull request merged**